timetracker/timetracker/urls.py

```diff
@@ -19,7 +19,20 @@
 
 
 def hello_world(request):
-    return HttpResponse('Hello World')
+    blank_html='''
```

Comment:
this should include a doctype:

```
    blank_html='''<!DOCTYPE html>
```

Also remember - all newlines in multi-line strings are included, including the one you have immediately following the '''